src/CarCrashsim/Vector.java

```diff
@@ -0,0 +1,76 @@
+package CarCrashsim;
+
+public class Vector {
```

Comment:
Good work on this class! Note that you’ve duplicated what’s already in the `Point` class in kilt-graphics; I think every one of these methods is already there. Don’t be too eager to build your own when a library can do the job for you! That said, good work putting together your own version.